frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/PushApprovalDialog.kt

```diff
@@ -0,0 +1,21 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.ide.util.PropertiesComponent
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DoNotAskOption
+import com.intellij.openapi.ui.Messages
+import com.virtuslab.gitmachete.frontend.actions.traverse.TraverseAction
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString
+
+class PushApprovalDialog(val project: Project) : DoNotAskOption.Adapter() {
+
+  override fun rememberChoice(isSelected: Boolean, exitCode: Int) {
```

Comment:
actually, I do not think `rememberChoice` is a good idea for push. again, we need three options: Abort Traverse, Yes, No; and a question like from CLI: 
> Branch allow-ownership-link diverged from (and has newer commits than) its remote counterpart origin/allow-ownership-link.
Push allow-ownership-link with force-with-lease to origin?